tests/fields/test_history.py

```diff
@@ -481,6 +482,7 @@ def test_history_field(mock_record, mock_swimlane):
         for idx, revision in enumerate(history):
             assert isinstance(revision, Revision)
```

Comment:
The tests for this are now failing due to a mismatch between `$type` and the mocked type https://travis-ci.org/swimlane/swimlane-python/jobs/528976400